site/documentation/content/concepts/command-and-control.md

```diff
@@ -53,24 +53,28 @@ that was opened by the application. If the response reached the application, the
 
 When the device is connected to the MQTT Adapter it receives *Request/Response* commands on the topic:
 
-* `control/[${tenant}]/[${device-id}]/req/${req-id}/${command}`
+* `command/[${tenant}]/[${device-id}]/req/${req-id}/${command}`
 
 and *one-way* commands on the topic:
 
-* `control/[${tenant}]/[${device-id}]/req//${command}`
+* `command/[${tenant}]/[${device-id}]/req//${command}`
 
 Authenticated devices typically subscribe to
 
-* `control/+/+/req/#`
+* `command/+/+/req/#`
 
 while unauthenticated devices have to fully specify their `${tenant}` and `${device-id}` during the subscription.
 
 The response of the command will be sent by the device to 
 
-* `control/[${tenant}]/[${device-id}]/res/${req-id}/${status}`
+* `command/[${tenant}]/[${device-id}]/res/${req-id}/${status}`
 
 If the device is authenticated, the `${tenant}` and `${device-id}` are left empty (resulting in 3 subsequent `/`s).
 
+{{% note %}}
+Previous versions of Hono used `control` instead of `command` as topic prefix. Using the `control` prefix is still supported but deprecated. 
```

Comment:
I think we can omit this note from the concept page ... it is sufficient to have this in the API spec FMPOV

Reply:
Ok, I've removed it.